### PR TITLE
AQT: Improve documentation, Add Z operations and rename AQTRemoteSimulator to AQTSamplerLocalSimulator

### PR DESCRIPTION
**Pull request by pschindler**
The API for the AQT devices has been changed and should now be stable. The code for accessing the API has been changed and handling of connection errors has been improved.

This PR adds Z operations for the AQT device.

This PR addresses the issues #2194 #2091 

The documentation has been imroved, possible addressing #2503

### CODE REVIEW

File: cirq/aqt/aqt_device.py [hunk 1]
```diff
@@ -62,6 +65,15 @@ def __init__(self):
 
     def noisy_moment(self, moment: ops.Moment,
                      system_qubits: Sequence[ops.Qid]):
+        """Returns a list of noisy moments including
```
Comment (vtomole) [inline cirq/aqt/aqt_device.py]: I keep reading `Depolarizing noise with gate-dependent strength Crosstalk  between neighboring qubbits` as a sentence which is confusing me. Also there is a typo for `qubits` here.

File: cirq/aqt/aqt_device.py [hunk 2]
```diff
@@ -24,19 +24,21 @@
...
 
 
-def get_op_string(op_obj: ops.Operation):
+def get_op_string(op_obj: ops.Operation) -> str:
     """Find the string representation for a given gate
-    Params:
-        op_obj: Gate object, out of: XXPowGate, XPowGate, YPowGate"""
+    Args:
+        op_obj: Gate object, one of: XXPowGate, XPowGate, YPowGate, ZPowGate
+    Returns:
```
Comment (Strilanc) [inline cirq/aqt/aqt_device.py]: Nit: Blank line between sections, `"""` on new line.

File: cirq/aqt/aqt_device.py [hunk 3]
```diff
@@ -61,7 +66,16 @@ def __init__(self):
         self.noise_op_dict = get_default_noise_dict()
 
     def noisy_moment(self, moment: ops.Moment,
-                     system_qubits: Sequence[ops.Qid]):
+                     system_qubits: Sequence[ops.Qid]) -> List[ops.Operation]:
+        """Returns a list of noisy moments. The model incluldes
```
Comment (Strilanc) [inline cirq/aqt/aqt_device.py]: nit: `includes` typo

File: cirq/aqt/aqt_device.py [hunk 4]
```diff
@@ -75,22 +89,29 @@ def noisy_moment(self, moment: ops.Moment,
         return list(moment) + noise_list
 
     def get_crosstalk_operation(self, operation: ops.Operation,
-                                system_qubits: Sequence[ops.Qid]):
+                                system_qubits: Sequence[ops.Qid]
+                               ) -> List[ops.Operation]:
         """
-        Returns operation on
+        Returns a list of operations including crosstalk
```
Comment (Strilanc) [inline cirq/aqt/aqt_device.py]: Nit: summary line goes on same line as `"""`

File: cirq/aqt/aqt_device.py [hunk 5]
```diff
@@ -127,19 +155,20 @@ def __init__(self,
...
         self.circuit = Circuit()
-        # TODO add ion device here, is this still required?
         json_obj = json.loads(json_string)
         for gate_list in json_obj:
             gate = gate_list[0]
             angle = gate_list[1]
             qubits = [self.qubit_list[i] for i in gate_list[2]]
             self.circuit.append(gate_dict[gate].on(*qubits)**angle)
-        # TODO: Better solution for measurement at the end
+        #TODO: Better solution for measurement at the end
```
Comment (Strilanc) [inline cirq/aqt/aqt_device.py]: Nit: keep the space.

Reference a person or issue URL to track the TODO.

**Comment by pschindler** [comment]
There seems to be an issue with the Travis test. The pytest-changed-files produces following error:

> ========================= no tests ran in 0.00 seconds =========================
> ERROR: file not found: cirq/aqt/aqt_sampler_test.py
> cirq/aqt/aqt_simulator_test.py
> The command "check/pytest-changed-files" exited with 4.

https://travis-ci.com/quantumlib/Cirq/jobs/258921745

**Comment by pschindler** [comment]
There seems to be an issue with  `check/pytest-changed-files`

If I remove the " from the last lines, it seems to work:

> pytest ${rest} ${changed[@]}

Should I open an issue for this?

**Comment by vtomole** [comment]
Merged in https://github.com/quantumlib/Cirq/pull/2584 and into your branch. That check passes here now.

File: cirq/aqt/aqt_device.py (hunk 3 above)
Comment (vtomole) [reply cirq/aqt/aqt_device.py]: @pschindler This typo is not fixed.

Comment (vtomole) [reply cirq/aqt/aqt_device.py]: Oops. I was reading an outdated version of the file.

**Pull request merged**